server/CONTRIBUTING.md

```diff
@@ -16,6 +16,17 @@ own machine and how to contribute.
 The last two prerequisites can be installed on Debian with:
 
     $ sudo apt install libpq-dev python3 python3-pip python3-venv
+    
+### Haskell + HIE Devcontainer 
+
+VS Code provides the ability to develop applications inside of a Docker container (called _Devcontainers_)
+https://code.visualstudio.com/docs/remote/containers
```

Comment:
I am reluctant to add information to the CONTRIBUTING.md that involves a workflow that nobody on the server team actually uses and therefore cannot meaningfully support. However, I believe @0x777 *does* have a workflow that he uses involving HIE; maybe he can speak to that here?